OJP/OJP_Trips.xsd

```diff
@@ -1117,6 +1118,27 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
+	<xs:group name="InterchangeGroup">
+		<xs:annotation>
+			<xs:documentation>Adding interchange elements from SIRI to a transfer leg.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="InterchangeCode" type="siri:InterchangeCodeType" minOccurs="0"/>
+			<xs:element name="InterchangeRef" type="siri:InterchangeRefStructure" minOccurs="0"/>
```

Comment:
@ue71603 What are InterchangeCode/Ref for, where do they come from?

Reply:
From Siri. I will remove the InterchangeCode (shouldn't have used it. If the interchanges have an id then InterchangeRef could be used to reference it. Needs either NeTEx or SIRI data about interchanges.